starknet_devnet/blueprints/rpc/transactions.py

```diff
@@ -123,7 +123,6 @@ async def add_declare_transaction(
     """
     if int(declare_transaction["version"], 0) == LEGACY_TX_VERSION:
         raise RpcError.from_spec_name("INVALID_CONTRACT_CLASS")
-
```

Comment:
I notice in a lot of places you delete some of the empty lines. What is your motivation for that?